### PR TITLE
[dom-gpu] Add missing CPMD and Amber

### PR DESCRIPTION
**Pull request by victorusu**

### CODE REVIEW

File: easybuild/easyconfigs/a/Amber/Amber-18-14-14-CrayGNU-19.06-cuda-10.0.eb [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# @author: victor holanda rusu
+
+easyblock = 'MakeCp'
+
+name = 'Amber'
+patchlevels = (14, 14) # (AmberTools, Amber)
+version = '18-%s-%s' % (patchlevels[0], patchlevels[1])
+cudaversion = '10.0'
```
Comment (teojgo) [inline easybuild/easyconfigs/a/Amber/Amber-18-14-14-CrayGNU-19.06-cuda-10.0.eb]: This is mentioned with cuda 10.1 in the production list

File: easybuild/easyconfigs/a/Amber/Amber-18-14-14-CrayGNU-19.06-cuda-10.0-pat-7.0.3.eb [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# @author: victor holanda rusu
+
+easyblock = 'MakeCp'
+
+name = 'Amber'
+patchlevels = (14, 14) # (AmberTools, Amber)
+version = '18-%s-%s' % (patchlevels[0], patchlevels[1])
+cudaversion = '10.0'
```
Comment (gppezzi) [inline easybuild/easyconfigs/a/Amber/Amber-18-14-14-CrayGNU-19.06-cuda-10.0-pat-7.0.3.eb]: cuda 10.1?

File: easybuild/easyconfigs/a/Amber/Amber-18-14-14-CrayGNU-19.06-cuda-10.1-pat-7.0.3.eb [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# @author: victor holanda rusu
+
+easyblock = 'MakeCp'
+
+name = 'Amber'
+patchlevels = (14, 14) # (AmberTools, Amber)
+version = '18-%s-%s' % (patchlevels[0], patchlevels[1])
+cudaversion = '10.0'
```
Comment (gppezzi) [inline easybuild/easyconfigs/a/Amber/Amber-18-14-14-CrayGNU-19.06-cuda-10.1-pat-7.0.3.eb]: cuda 10.1?

Comment (victorusu) [reply easybuild/easyconfigs/a/Amber/Amber-18-14-14-CrayGNU-19.06-cuda-10.1-pat-7.0.3.eb]: Lesson learnt... Never do git add Amber-18-14-14-CrayGNU-19.06-cuda-*.eb

File: easybuild/easyconfigs/a/Amber/Amber-18-14-14-CrayGNU-19.06-cuda-10.0-pat-7.0.3.eb (hunk 2 above)
Comment (victorusu) [reply easybuild/easyconfigs/a/Amber/Amber-18-14-14-CrayGNU-19.06-cuda-10.0-pat-7.0.3.eb]: Lesson learnt... Never do git add Amber-18-14-14-CrayGNU-19.06-cuda-*.eb

**Pull request merged**